pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -0,0 +1,63 @@
+
+__all__ = [
+    'Queue'
+]
+
+class Queue(object):
+    """Respresentation of queue data structure
+
+    Examples
```

Comment:
Please change to `Examples` in [this line](https://github.com/codezonediitj/pydatastructs/blob/master/pydatastructs/miscellaneous_data_structures/stack.py#L32) as well.